packages/babel-plugin-jest-hoist/package.json

```diff
@@ -20,10 +20,11 @@
     }
   },
   "dependencies": {
+    "@babel/template": "^7.3.3",
```

Comment:
just to verify, these are new enough to work for us?

Reply:
I haven't made the Babel PR I wanted yet, so I think all APIs used are pretty standard, yeah